README.md

```diff
@@ -1,3 +1,80 @@
 # Magento 1 - Url Cache Control
 
-[will follow]
\ No newline at end of file
+Magento 1.x Extension
+
+## Overview
+
+This module appends the modified timestamp to skin url's, to automatically burst Browser Cache on changes.
```

Comment:
Should be "URLs" instead of "url's" and "bust" instead of "burst" ;).

What exactly is the difference to https://github.com/fbrnc/Aoe_JsCssTstamp?

Reply:
You are right. 

The aoe module append/merge the timestamp in the Filename and Not as Parameter.

So aoe generates bin existing URLs and need to rewrite that in htaccess. Also it is much more Code and copies produced.

Reply:
-bin +non

... Existing url's.

Sry Smartphone has own will :D